doc/fornax_setup.md

```diff
@@ -0,0 +1,245 @@
+
+```script
+sudo apt-get update
+sudo apt install build-essential
+sudo snap install curl  # version 7.81.0
+sudo apt-get install vim
+```
+
+## 1. Install and Setup Fornax Core
+### 1.1 Install Etcd 
+Install etcd by using apt-get
+```script
+sudo apt-get update
+sudo apt-get -y install etcd
+```
+
+Verify etcd
+```script
+sudo systemctl status etcd.service
+sudo systemctl enable etcd
+sudo systemctl restart etcd
+```
+
+Stop etcd.service. (note: we use the local etcd by default fornax serverless, so we need stop etcd.service)
+```script
+sudo systemctl stop etcd.service
+```
+
+### 1.2 Install Fornax Core From Source Code
+#### 1.2.1 Install Golang
+```script
+GOLANG_VERSION=${GOLANG_VERSION:-"1.18"}
+sudo apt -y update
+sudo apt -y install make
+sudo apt -y install gcc
+sudo apt -y install jq
+wget https://dl.google.com/go/go${GOLANG_VERSION}.linux-amd64.tar.gz -P /tmp
+sudo tar -C /usr/local -xzf /tmp/go${GOLANG_VERSION}.linux-amd64.tar.gz
+```
+veify go version
+```script
+export PATH=$PATH:/usr/local/go/bin
+go version
+```
+the result should be: go version go1.18 linux/amd64
+
+make following directory (under yourself home directory)
+```script
+sudo mkdir -p go/bin
+sudo mkdir -p go/src
+```
+
+Add following setting to machine ~/.bashrc (notes: ubuntu is yourself user login directory, remember to replace your's)
+```script
+sudo vi ~/.bashrc
+```
+
+```sh
+export GOPATH="/home/ubuntu/go"
+export GOROOT="/usr/local/go"
+export GOBIN="/home/ubuntu/go/bin"
+export PATH="$PATH:/usr/local/go/bin:/home/ubuntu/go/bin"
+```
+Take effetive
+```script
+sudo source ~/.bashrc
+```
+
+#### 1.2.2 Download and Compile Source Code
+Create working folder, download code, and go working folder (fornax-serverles)
+```script
+sudo mkdir -p src/centaurusinfra.io
+cd src/centaurusinfra.io
+```
+In centaurusinfra.io folder and run follwing command
+```script
+git clone https://github.com/CentaurusInfra/fornax-serverless.git
+cd fornax-serverless
+```
+You are in your workplace folder: fornax-serverless
+
+```script
+sudo make all
+```
+Now, you can back to ![get_start.md](get_start.md) and continue next step.
+
+
+
+## 2. Install and Setup Fornax Node Agent
+### 2.1 Install containerd/cni/runc
+#### 2.1.1 Install containerd
+Run following command
+```script
+sudo curl -fsSL https://get.docker.com -o get-docker.sh
+sudo sh get-docker.sh
+```
+
+Verify containerd containerd service is runing
+```script
+sudo systemctl restart containerd
+sudo systemctl status containerd
+```
+Notes: Fornax-serverless using containerd.service. So you need stop docker.service.
+```script
+sudo systemctl status docker
+sudo systemctl stop docker
+sudo systemctl status docker
+```
+
+#### 2.1.2 Enable Containerd CRI Plugins
+
+Find containerd config.toml file
+```script
+dpkg -L containerd.io | grep toml
+```
+Using vi to open the file
```

Comment:
why must vi, people can use any editor they like to use

Reply:
You are right, I can change to "You can use vi or  other tool to open the file"